File: 프로그래머스/unrated/161990. 바탕화면 정리/바탕화면 정리.java
import java.util.*;
class Solution {
    public int[] solution(String[] wallpaper) {
        int[] answer = {};
        ArrayList<int[]> files = new ArrayList<>();
        
        for(int i=0;i<wallpaper.length;i++){
            if(wallpaper[i].contains("#")){
                for(int j=0;j<wallpaper[i].length();j++){
                    if(wallpaper[i].charAt(j)=='#')
                        files.add(new int[]{i,j});
                }
                
            }
        }
        int maxr=0, maxc=0, minr=50, minc=50;
        for(int[] a: files){
            maxr=Math.max(maxr,a[0]);
            maxc=Math.max(maxc,a[1]);
            minr=Math.min(minr,a[0]);
            minc=Math.min(minc,a[1]);
        }
        answer = new int[]{minr,minc,maxr+1,maxc+1};
        return answer;
    }
}

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    answer = []
    rdx,rdy,lux,luy=0,0,49,49
    
    for i in range(len(wallpaper)):
        for j in range(len(wallpaper[i])):
            if wallpaper[i][j]=='#':
                lux=min(i,lux)
                luy=min(j,luy)
                rdx=max(i,rdx)
                rdy=max(j,rdy)

    return [lux,luy,rdx+1,rdy+1]